File: hello/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (!(n >= 1 && n <= 8));

    int s = n - 1;

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < s; j++)
        {
            printf(" ");
        }

        for (int k = 0; k < n - s; k++)
        {
            printf("#");
        }

        printf("  ");

        for (int k = 0; k < n - s; k++)
        {
            printf("#");
        }

        s--;

        printf("\n");
    }
}

File: sentimental-hello/sentimental-mario-less/mario.py
from cs50 import get_int

h = get_int("Height: ")
while (h <= 0 or h >= 9):
    h = get_int("Height: ")

for i in range(h):
    for j in range(h-i-1):
        print(" ", end="")
    for j in range(i+1):
        print("#", end="")

    print()
